.github/workflows/cd.yml

```diff
@@ -29,8 +29,8 @@ jobs:
       - name: Test
         run: npm run test
 
-      - name: Deploy test report
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
+      # - name: Deploy test report
```

Comment:
would prefer to remove or to give some context as to why this step is commented out

Reply:
because we don't need to build a report for the private fork we are using to test the vdp.